setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import Command, find_packages, setup
 import os
 
-VERSION = '0.58.3'
+VERSION = '0.59.0'
```

Comment:
compat is still alpha stage.. so it will stay as major v0.

I am not happy with the API ... the representation of paths as segments is a mess.

But maybe we will do a v1 release once it is ported to py3 ... and update to API to something similar to py3 pathlib  https://docs.python.org/3/library/pathlib.html